gateways/core/webhook/webhook.go

```diff
@@ -200,13 +200,17 @@ func (wce *webhookConfigExecutor) StopConfig(config *gateways.ConfigContext) err
 }
 
 func main() {
-	_, err := gatewayConfig.WatchGatewayEvents(context.Background())
+	err := gatewayConfig.TransformerReadinessProbe()
```

Comment:
is there any way to reduce some of this boilerplate code (possibly centralize to a util method).. I'm seeing this code and corresponding changes for all gateway types...